### ISSUE TITLE
newevents.js is not processing events that repeat

### ISSUE THREAD

**Issue by valentine** [hackerspacesg/hackerspace.sg#148]
[newevents.js](https://github.com/hackerspacesg/hackerspace.sg/blob/master/bin/newevents.js) isn't processing events that repeat regularly. The `Cappella Martialis Rehearsal` events are missing from the events list.

**Comment by valentine** [comment]
Ping @kaihendry, @notthetup, and @chernjie.

**Comment by notthetup** [comment]
@chernjie Remind me again, where this bit runs? It's on azure?

**Comment by valentine** [comment]
It's run directly on Netlify now. Doesn't seem to process the repeating events locally with `node bin/newevents.js` though.

**Comment by notthetup** [comment]
Right. Maybe it's an issue with repeating events. I'll take a look at it.
